hibp/table_hibp_password.go

```diff
@@ -37,7 +37,8 @@ func tableHIBPPassword() *plugin.Table {
 }
 
 func listPasswords(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	client, err := getHibpClient(ctx, d)
+
+	client, err := getHibpClientWithoutAPIKey(ctx, d)
```

Comment:
Are the rate limits different if we authenticate without an API key? Are there any other differences or advantages of using an API key, even if the API doesn't require one?

Reply:
Referring [to this part of the docs](https://haveibeenpwned.com/API/v3#PwnedPasswords):
> There is no rate limit on the Pwned Passwords API. 

Referring to [this part of the docs](https://haveibeenpwned.com/API/v3#RateLimiting)
> Requests to the breaches and pastes APIs are limited. The rate limits depends on the [the API key you've purchased](https://haveibeenpwned.com/API/Key).

I can't find mention of what the different rate limits are per the different keys.